openpoiservice/tests/test_geocoder.py

```diff
@@ -0,0 +1,89 @@
+# openpoiservice/server/tests/test_main.py
+
+
+import unittest
+import json
+from openpoiservice.server.db_import.objects import AddressObject, GeocoderSetup
+from openpoiservice.server.categories.categories import CategoryTools
+
+# from base import BaseTestCase
```

Comment:
Cannot find references 'BaseTestCase'. Same in other testing files. Therefore, line 10 "from openpoiservice.tests.base import BaseTestCase"

Reply:
The folder structure for testing the osm files is invalid. See "/openpoiservice/tests/base.py#L21"